module/apmgopg/hook_test.go

```diff
@@ -0,0 +1,68 @@
+package apmgopg_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-pg/pg"
+	"github.com/go-pg/pg/orm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"go.elastic.co/apm/apmtest"
+)
+
+type User struct {
+	TableName struct{} `sql:"users"`
+
+	Id   int    `json:"id"`
+	Name string `json:"name"`
+}
+
+func TestWithContext(t *testing.T) {
+
+	_, spans, errors := apmtest.WithTransaction(func(ctx context.Context) {
+		db := pg.Connect(&pg.Options{
+			Addr:     "localhost:5432",
```

Comment:
You should use the $PGHOST env var here, if it's supplied, like in the `apmsql/pq` integration test:

https://github.com/elastic/apm-agent-go/blob/e4cbc85a0ea49329a88055d657e5a3892761e683/module/apmsql/pq/pq_test.go#L35-L37

Note that `lib/pq` uses the PGHOST env var if you don't specify a host in the DSN passed to sql.Open, which is why you don't see the value used in that code. You might need to pass the value into the options here.